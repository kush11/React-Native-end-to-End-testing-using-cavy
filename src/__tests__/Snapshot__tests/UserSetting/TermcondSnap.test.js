import 'react-native';
import React from 'react';
import Renderer from 'react-test-renderer';
import TermsCondition from '../../../screens/userSettings/TermsConditions';

test('Terms snapshot', () => {
  const snap = Renderer.create(
    <TermsCondition />
  ).toJSON();
  expect(snap).toMatchSnapshot();
});
