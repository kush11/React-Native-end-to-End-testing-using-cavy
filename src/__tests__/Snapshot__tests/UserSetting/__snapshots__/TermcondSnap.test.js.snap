// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Terms snapshot 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "black",
        "fontFamily": "OpenSans-Regular",
        "fontSize": 18,
        "textAlign": "center",
      }
    }
  />
</View>
`;
